package.json

```diff
@@ -59,13 +66,15 @@
     "eslint-plugin-ember": "^5.0.0",
     "eslint-plugin-node": "^6.0.1",
     "loader.js": "^4.2.3",
+    "mocha": "^5.0.0",
     "qunit-dom": "^0.7.1",
     "typescript": "^2.9.2"
   },
   "engines": {
     "node": "6.* || 8.* || >= 10.*"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "defaultBlueprint": "install-sparkles"
```

Comment:
this is my best idea for avoiding the `sparkles-component` blueprint being run as a package installation blueprint. `install-sparkles` is a no-op placeholder.

Reply:
Yep, exactly what I was thinking.

https://github.com/ember-cli/ember-cli/blob/9d7fd5f3fea476944a4e9ddc38f1f7da57fee0ca/lib/tasks/addon-install.js#L85-L87